cluster-operator/src/test/java/io/strimzi/operator/cluster/model/AbstractConfigurationTest.java

```diff
@@ -238,6 +238,16 @@ public void testKafkaZookeeperTimeout() {
         assertThat(kc.asOrderedProperties().asMap().get("zookeeper.connection.timeout"), is(nullValue()));
     }
 
+    @Test
+    public void testKafkaCipherSuiteOverride() {
+        Map<String, Object> conf = new HashMap<>();
+        conf.put("ssl.cipher.suites", "cipher1,cipher2,cipher3"); // valid
+
+        KafkaConfiguration kc = new KafkaConfiguration(conf.entrySet());
+
+        assertThat(kc.asOrderedProperties().asMap().get("ssl.cipher.suites"), is("cipher1,cipher2,cipher3"));
```

Comment:
Hi John, from a test debugging perspective, the below might be a little bit cleaner:
```
assertThat(kc.asOrderedProperties().asMap(), hasEntry("ssl.cipher.suites", "cipher1,cipher2,cipher3"));
```
Where you statically import hasEntry with: `import static org.hamcrest.Matchers.hasEntry;`

Reply:
Hi Sam, this code is consistent with the other tests on which is was based, so I would suggest this change be left as is and then if its desirable to modify them, we do all of them together in another PR. Just my thoughts on that one...

Reply:
Perfectly reasonable John!